src/app/services/navigation/navigation.service.ts

```diff
@@ -149,13 +151,16 @@ export class NavigationService {
       return;
     }
 
-    this.store$.pipe(waitForSystemInfo, untilDestroyed(this))
-      .subscribe((systemInfo) => {
-        const hasVms = systemInfo.license && Boolean(systemInfo.license.features.includes(LicenseFeature.Vm));
-        this.hasVms$.next(hasVms);
-
-        const hasApps = systemInfo.license && Boolean(systemInfo.license.features.includes(LicenseFeature.Jails));
-        this.hasApps$.next(hasApps);
-      });
+    this.store$.pipe(
+      waitForSystemInfo,
+      filter((systemInfo) => Boolean(systemInfo.license)),
+      map((systemInfo) => systemInfo.license.features),
+      untilDestroyed(this),
+    ).subscribe((features) => {
+      this.hasVms$.next(features.includes(LicenseFeature.Vm));
```

Comment:
I think something happened to `hasVms$` check here at some point because Instances menu item is not disabled when it is false.